python/dask_cudf/dask_cudf/sorting.py

```diff
@@ -294,6 +294,6 @@ def sort_values(
     df4 = df3.map_partitions(sort_function, **sort_kwargs)
     if not isinstance(divisions, gd.DataFrame) and set_divisions:
         # Can't have multi-column divisions elsewhere in dask (yet)
-        df4.divisions = methods.tolist(divisions)
+        df4.divisions = tuple(methods.tolist(divisions))
```

Comment:
This fix is required due to this upstream change: https://github.com/dask/dask/pull/8806